profile/Client/profileClient.go

```diff
@@ -0,0 +1,318 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+// profileclient to handle profiling of kubearmor telemetry events
+package profileclient
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/charmbracelet/bubbles/help"
+	"github.com/charmbracelet/bubbles/key"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/evertras/bubble-table/table"
+	pb "github.com/kubearmor/KubeArmor/protobuf"
+	klog "github.com/kubearmor/kubearmor-client/log"
+	profile "github.com/kubearmor/kubearmor-client/profile"
+)
+
+const (
+	columnNamespace   = "namespace"
+	columnPodname     = "podname"
+	columnProcessName = "ProcName"
+	columnResource    = "Resource"
+	columnResult      = "Result"
+	columnCount       = "count"
+	columnTimestamp   = "timestamp"
```

Comment:
Are these strings column names? We should either keep them all CamelCase or SmallCase for uniformity.

Reply:
not exactly, these are the "keys" to which a column name is assigned when making a table in bubble tea. And a key is unique to that table column, hence should be a constant with any string of fixed value. This Key communicates between the columns made, and what values should each key get when making a new row.

But yeah I'll make them CamelCase